src/main/resources/templates/navbar.html

```diff
@@ -8,16 +8,12 @@
     <div class="container d-flex flex-column flex-md-row justify-content-between">
         <div class="col-sm-12 col-md-6 col-lg-2"><a href="/">home</a></div>
         <div class="col-sm-12 col-md-6 col-lg-2"><a href="/grades">grades</a></div>
-        <div class="col-sm-12 col-md-6 col-lg-2"><a href="/chat">chat</a></div>
+ <!--       <div class="col-sm-12 col-md-6 col-lg-2"><a href="/chat">chat</a></div>-->
```

Comment:
delete or keep - but please do not comment out

Reply:
done